runtime/moonbeam/src/lib.rs

```diff
@@ -1274,6 +1274,8 @@ construct_runtime! {
 		XTokens: orml_xtokens::{Pallet, Call, Storage, Event<T>} = 106,
 		XcmTransactor: pallet_xcm_transactor::{Pallet, Call, Storage, Event<T>} = 107,
 		LocalAssets: pallet_assets::<Instance1>::{Pallet, Call, Storage, Event<T>} = 108,
+		// Reserving id 109 for future integration of pallet_erc20_xcm_bridge
+		//Erc20XcmBridge: pallet_erc20_xcm_bridge::{Pallet} = 109,
```

Comment:
If you want to be really pedantic, you could write a test to cover this :)

Reply:
We don't need to reserve pallet id for moonriver/moonbeam, I just theses comments